showcase/README.md

```diff
@@ -78,6 +79,10 @@ Note:
 Open a new terminal window in the root project directory.
 
 ```shell
-$ cd showcase
-$ mvn compile -P update
+# Likely you need to reinstall gapic-generator-java 
+# before updating the showcase.
+# In repository's root directory
+mvn clean install -DskipTests
```

Comment:
It should be outside of the scope of this PR, but just want to bring up ideas for possible enhancement. I _think_ if we make showcase a submodule of the repo (compare to a standalone module right now), then we should be able to run showcase tests/update goldens without re-installing the generator, but it may have other implications. @suztomo may have more ideas about it.

Reply:
I think that idea will work. We just have to be careful that the showcase artifact won't be released to Maven Central.